api/s2n.h

```diff
@@ -2721,6 +2721,13 @@ extern int s2n_connection_client_cert_used(struct s2n_connection *conn);
 /**
  * A function that provides a human readable string of the cipher suite that was chosen
  * for a connection.
+ *
+ * @warning The string "TLS_NULL_WITH_NULL_NULL" is returned before the TLS handshake has been performed.
+ * This does not mean that the ciphersuite "TLS_NULL_WITH_NULL_NULL" will be used by the connection,
+ * it is merely being used as a placeholder.
```

Comment:
Are we adding documentation instead of fixing the behavior to avoid breaking current customers somehow?

Reply:
Oh wait, just saw https://github.com/aws/s2n-tls/issues/2766 :)